libdlib/CMakeLists.txt

```diff
@@ -16,8 +16,8 @@ ExternalProject_Add(EP_${PROJECT_NAME}
     SOURCE_DIR ${PROJECT_BINARY_DIR}/${PROJECT_NAME}-src
     BINARY_DIR ${PROJECT_BINARY_DIR}/${PROJECT_NAME}-build
     
-    CONFIGURE_COMMAND ${CMAKE_COMMAND} ${PROJECT_BINARY_DIR}/${PROJECT_NAME}-src/dlib
-    BUILD_COMMAND ${CMAKE_COMMAND} --build . --config Release -- -Wno-deprecated-declarations -Wno-terminate
+    CONFIGURE_COMMAND ${CMAKE_COMMAND} CFLAGS=-Wno-deprecated-declarations\ -Wno-terminate CXXFLAGS=-Wno-deprecated-declarations\ -Wno-terminate ${PROJECT_BINARY_DIR}/${PROJECT_NAME}-src/dlib
+    BUILD_COMMAND ${CMAKE_COMMAND} --build . --config Release
```

Comment:
@ipa-rmb @ipa-foj @ipa-fog @ipa-jcl 
this does not get rid of the warnings under `melodic` - is there a new version of `libdlib` that would fix this?

Reply:
I do not know, did not use it for a while. Did you use it, @ipa-foj? Do you know whether we could just use a newer version?

Reply:
see #104